persistence/src/main/scala/hmda/persistence/institutions/InstitutionPersistence.scala

```diff
@@ -45,31 +48,35 @@ class InstitutionPersistence extends HmdaPersistentActor {
 
   override def receiveCommand: Receive = {
     case CreateInstitution(i) =>
-      if (!state.institutionIds.contains(i.id)) {
+      if (!state.institutions.contains(i)) {
```

Comment:
Good question. I don't think it should be created if it has the same id. I'm going to go back to checking creation by making sure the id is not already present